roles/swift-ring/tasks/main.yml

```diff
@@ -13,3 +13,4 @@
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
   when: not should_boostrap|success
+  notify: restart swift services
```

Comment:
Currently relevant services are started after each swift component is installed (proxy, container, etc). We can move around roles but that could be error prone due to dependencies. Also the flow that we have right now (install all components, build rings, start services) makes logical sense and recommended one. Is there a reason you don't like restarting services when rings are deployed?